### ISSUE TITLE
cli has 2 vulnerabilities

### ISSUE THREAD

**Issue by hgezim** [strongloop/loopback#4150]
# Description/Steps to reproduce

- Run `npm i  @loopback/cli --save-dev`

NPM reports 2 vulnerabilities. They'll be fixed by upgrading to lodash >=4.17.11.

# Link to reproduction sandbox

https://www.npmjs.com/advisories/577
https://www.npmjs.com/advisories/782

# Expected result
 
0 vulnerabilities

# Additional information

**Comment by goranovs** [comment]
There are 3 vulnerabilities with the latest version.

```
  Moderate        Prototype Pollution
  Package         lodash
  Patched in      >=4.17.11
  Dependency of   @loopback/build [dev]
  Path            @loopback/build > strong-docs > dox > jsdoctypeparser > lodash
  More info       https://nodesecurity.io/advisories/782

  Low             Prototype Pollution
  Package         lodash
  Patched in      >=4.17.5
  Dependency of   @loopback/build [dev]
  Path            @loopback/build > strong-docs > dox > jsdoctypeparser > lodash
  More info       https://nodesecurity.io/advisories/577


  Moderate        Denial of Service
  Package         js-yaml
  Patched in      >=3.13.0
  Dependency of   @loopback/build [dev]
  Path            @loopback/build > mocha > js-yaml
  More info       https://nodesecurity.io/advisories/788
```

**Comment by shilpank** [comment]
We are seeing the below vulnerability with the loopback latest version :

lodash": "^4.14.151"


Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/boot                                                

  Path            @loopback/boot > @loopback/service-proxy >                    
                  loopback-datasource-juggler > strong-globalize > lodash       

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/service-proxy                                       

  Path            @loopback/service-proxy > loopback-datasource-juggler >       
                  strong-globalize > lodash                                     

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/rest                                                

  Path            @loopback/rest > strong-error-handler > strong-globalize >    
                  lodash                                                        

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/rest-explorer                                       

  Path            @loopback/rest-explorer > @loopback/rest >                    
                  strong-error-handler > strong-globalize > lodash              

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/openapi-v3                                          

  Path            @loopback/openapi-v3 > lodash                                 

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/rest                                                

  Path            @loopback/rest > @loopback/openapi-v3 > lodash                

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/rest-explorer                                       

  Path            @loopback/rest-explorer > @loopback/rest >                    
                  @loopback/openapi-v3 > lodash                                 

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/rest                                                

  Path            @loopback/rest > json-schema-compare > lodash                 

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/rest-explorer                                       

  Path            @loopback/rest-explorer > @loopback/rest >                    
                  json-schema-compare > lodash                                  

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/rest                                                

  Path            @loopback/rest > lodash                                       

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/rest-explorer                                       

  Path            @loopback/rest-explorer > @loopback/rest > lodash             

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   csvtojson                                                     

  Path            csvtojson > lodash                                            

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   lodash                                                        

  Path            lodash                                                        

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   loopback-connector-rest                                       

  Path            loopback-connector-rest > lodash                              

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   loopback-connector-rest                                       

  Path            loopback-connector-rest > strong-globalize > lodash           

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   winston                                                       

  Path            winston > async > lodash                                      

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > @typescript-eslint/eslint-plugin >          
                  @typescript-eslint/experimental-utils >                       
                  @typescript-eslint/typescript-estree > lodash                 

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @typescript-eslint/eslint-plugin                              

  Path            @typescript-eslint/eslint-plugin >                            
                  @typescript-eslint/experimental-utils >                       
                  @typescript-eslint/typescript-estree > lodash                 

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > @typescript-eslint/parser >                 
                  @typescript-eslint/experimental-utils >                       
                  @typescript-eslint/typescript-estree > lodash                 

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @typescript-eslint/parser                                     

  Path            @typescript-eslint/parser >                                   
                  @typescript-eslint/experimental-utils >                       
                  @typescript-eslint/typescript-estree > lodash                 

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > @typescript-eslint/parser >                 
                  @typescript-eslint/typescript-estree > lodash                 

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @typescript-eslint/parser                                     

  Path            @typescript-eslint/parser >                                   
                  @typescript-eslint/typescript-estree > lodash                 

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > eslint > inquirer > lodash                  

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   eslint                                                        

  Path            eslint > inquirer > lodash                                    

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > eslint > lodash                             

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   eslint                                                        

  Path            eslint > lodash                                               

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > eslint > table > lodash                     

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   eslint                                                        

  Path            eslint > table > lodash                                       

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > mocha > yargs-unparser > lodash             

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/generator > @babel/types > lodash        

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helper-module-transforms >               
                  @babel/helper-replace-supers > @babel/traverse >              
                  @babel/generator > @babel/types > lodash                      

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helpers > @babel/traverse >              
                  @babel/generator > @babel/types > lodash                      

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/traverse > @babel/generator >            
                  @babel/types > lodash                                         

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helper-module-transforms >               
                  @babel/helper-module-imports > @babel/types > lodash          

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helper-module-transforms >               
                  @babel/helper-replace-supers >                                
                  @babel/helper-member-expression-to-functions > @babel/types   
                  > lodash                                                      

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helper-module-transforms >               
                  @babel/helper-replace-supers >                                
                  @babel/helper-optimise-call-expression > @babel/types >       
                  lodash                                                        

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helper-module-transforms >               
                  @babel/helper-replace-supers > @babel/traverse >              
                  @babel/helper-function-name >                                 
                  @babel/helper-get-function-arity > @babel/types > lodash      
        
  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helpers > @babel/traverse >              
                  @babel/helper-function-name >                                 
                  @babel/helper-get-function-arity > @babel/types > lodash      

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/traverse > @babel/helper-function-name   
                  > @babel/helper-get-function-arity > @babel/types > lodash    

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helper-module-transforms >               
                  @babel/helper-replace-supers > @babel/traverse >              
                  @babel/helper-function-name > @babel/template > @babel/types  
                  > lodash                                                      

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helpers > @babel/traverse >              
                  @babel/helper-function-name > @babel/template > @babel/types  
                  > lodash                                                      

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/traverse > @babel/helper-function-name   
                  > @babel/template > @babel/types > lodash                     

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helper-module-transforms >               
                  @babel/helper-simple-access > @babel/template > @babel/types  
                  > lodash                                                      

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helper-module-transforms >               
                  @babel/template > @babel/types > lodash                       

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helpers > @babel/template >              
                  @babel/types > lodash                                         

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/template > @babel/types > lodash         

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helper-module-transforms >               
                  @babel/helper-replace-supers > @babel/traverse >              
                  @babel/helper-function-name > @babel/types > lodash           

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helpers > @babel/traverse >              
                  @babel/helper-function-name > @babel/types > lodash           

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/traverse > @babel/helper-function-name   
                  > @babel/types > lodash                                       

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helper-module-transforms >               
                  @babel/helper-replace-supers > @babel/traverse >              
                  @babel/helper-split-export-declaration > @babel/types >       
                  lodash                                                        

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helpers > @babel/traverse >              
                  @babel/helper-split-export-declaration > @babel/types >       
                  lodash                                                        

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/traverse >                               
                  @babel/helper-split-export-declaration > @babel/types >       
                  lodash                                                        

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helper-module-transforms >               
                  @babel/helper-split-export-declaration > @babel/types >       
                  lodash                                                        

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helper-module-transforms >               
                  @babel/helper-replace-supers > @babel/traverse >              
                  @babel/types > lodash                                         

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helpers > @babel/traverse >              
                  @babel/types > lodash                                         

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/traverse > @babel/types > lodash         

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helper-module-transforms >               
                  @babel/helper-replace-supers > @babel/types > lodash          

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helper-module-transforms >               
                  @babel/helper-simple-access > @babel/types > lodash           

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helper-module-transforms > @babel/types  
                  > lodash                                                      

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helpers > @babel/types > lodash          

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/types > lodash                           

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/generator > lodash                       

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helper-module-transforms >               
                  @babel/helper-replace-supers > @babel/traverse >              
                  @babel/generator > lodash                                     

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helpers > @babel/traverse >              
                  @babel/generator > lodash                                     

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/traverse > @babel/generator > lodash     

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helper-module-transforms >               
                  @babel/helper-replace-supers > @babel/traverse > lodash       

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helpers > @babel/traverse > lodash       

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/traverse > lodash                        

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > @babel/helper-module-transforms > lodash        

  More info       https://npmjs.com/advisories/1523                             


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      No patch available                                            

  Dependency of   @loopback/build                                               

  Path            @loopback/build > nyc > istanbul-lib-instrument >             
                  @babel/core > lodash                                          

  More info       https://npmjs.com/advisories/1523                             

found 131 low severity vulnerabilities in 832 scanned packages
  131 vulnerabilities require manual review. See the full report for details.
